### ISSUE TITLE
Shredder Does Not Drop Contents When Broken

### ISSUE THREAD

**Issue by EnderofGames** [GenDeathrow/Hatchery#125]
Using: 
 - Skyfactory 3 (3.0.15)
 - Hatchery 0.3.21

As title states. Any feathers, feather fibres, feather meal, and upgrades in the shredder disappear when the block is broken. 

RF is also lost, although that is arguably an enhancement, not a bug.

**Comment by GenDeathrow** [comment]
Well thanks for reminding me. I must have forgotten to add that in to drop items.

**Comment by IamAlathe** [comment]
Neither does the lucky egg machine in 1.12.2.  I don't know about the other machines (including shredder).

**Comment by GenDeathrow** [comment]
It should be dropping everything but eggs, and plastic. I will double check everything. and prolly add in ability to drop the eggs and plastic. in 1.12.2

**Comment by IamAlathe** [comment]
Yes, when broken it does not drop, nor retain eggs or plastic.  I've never broken it with lucky eggs in it, so it might drop those if there were some ready.  I wasn't sure if that was intentional or not.

**Comment by GenDeathrow** [comment]
its not technically intentional, but the eggs and plastic are just a number value and not an actual inventory. so when it breaks its empty. I'm looking into change that. It was just a lot easier to handle large numbers this way.

**Comment by IamAlathe** [comment]
Thanks for the explanation!  That makes a lot of sense.  Those materials are pretty easy to come by, so I wasn't too fussed.
